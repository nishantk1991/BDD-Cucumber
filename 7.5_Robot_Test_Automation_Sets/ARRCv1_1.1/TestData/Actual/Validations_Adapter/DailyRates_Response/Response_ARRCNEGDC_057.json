{
  "timestamp": "2024-11-22T05:43:34.104+00:00",
  "status": 400,
  "error": "Bad Request",
  "path": "/corporate/lending/alternative-reference-rates/v1/daily-compounding/rates"
}