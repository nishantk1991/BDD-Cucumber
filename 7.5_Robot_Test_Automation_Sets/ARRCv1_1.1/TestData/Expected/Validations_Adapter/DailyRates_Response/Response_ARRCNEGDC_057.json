{
  "message": "Bad Request",
  "_links": {
    "self": {
      "href": "/corporate/lending/alternative-reference-rates/v1/daily-compounding/rates?riskFreeRate=SOFR&rateRoundingDecimalPoints=2&lookBackDays=3&interestPeriodStartDate=2020-05-01&calculateTillDate=2020-05-15",
      "templated": false
    }
  },
  "_embedded": {
    "errors": [
      {
        "message": "Required QueryValue [interestPeriodEndDate] not specified",
        "path": "/interestPeriodEndDate"
      }
    ]
  }
}